### DISCUSSION TITLE
Midi feature

### DISCUSSION THREAD

**Discussion by kaloun34** [frntc/Sidekick64#55] [Q&A]
Hello
Is it possible to use the midi features without the SID wire ?
I can't get it to work

**Comment by kaloun34** [comment]
It's ok, I found how to do it

**Reply by odflor** [reply]
Please, how did You do it?

**Reply by frntc** [reply]
I noticed that I had a little bug -- once you also enable FM-emulation, then you can start programs with MIDI enabled (don't forget to copy soundfonts to the right folder/filename)
